docs/blog/gatekeeper-policies-as-oci-image.md

```diff
@@ -0,0 +1,215 @@
+# Bundle with ORAS, test with `gator` and deploy with GitOps your Gatekeeper policies as OCI image
```

Comment:
This title is a bit of a bag of words - maybe it's just me, but perhaps something shorter and more succinct?

Reply:
Good point, simplified it by `Bundle, test and deploy Gatekeeper policies as OCI image`